substrate/frame/conviction-voting/src/types.rs

```diff
@@ -32,18 +32,8 @@ use super::*;
 use crate::{AccountVote, Conviction, Vote};
 
 /// Info regarding an ongoing referendum.
-#[derive(
-	CloneNoBound,
-	PartialEqNoBound,
-	EqNoBound,
-	RuntimeDebugNoBound,
-	TypeInfo,
-	Encode,
-	Decode,
-	MaxEncodedLen,
-)]
-#[scale_info(skip_type_params(Total))]
-#[codec(mel_bound(Votes: MaxEncodedLen))]
+#[derive(CloneNoBound, PartialEqNoBound, EqNoBound, RuntimeDebugNoBound)]
+#[frame_support::stored(skip(Total), mel(Votes))]
 pub struct Tally<Votes: Clone + PartialEq + Eq + Debug + TypeInfo + Codec, Total> {
```

Comment:
The difficult part about this was that it tries to re-add the `Clone` bound for all parameters that are not listed here.  
And that requires some changes to all of our `*NoBound` derives. Now i am thinking that we can just scrap that and require the developer to write `Votes: Clone` themselves like we do here.

Reply:
if I read it right, `mel(Votes)` here means do not generate anything for `Votes` and use whatever is set for it below? wonder why `MaxEncodedLen` is not needed then?